exprconv/expr.hpp

```diff
@@ -58,4 +68,44 @@ class ExprtkConverter : public DataConverter {
             int dummy = (int)(val * divider);
             return dummy / divider;
         }
+
+        static double int32be(const double register1, const double register2) {
+            return ConverterTools::toNumber<int32_t>(register1, register2);
+        }
+
+        static double int32le(const double register1, const double register2) {
+            return ConverterTools::toNumber<int32_t>(register1, register2, true);
+        }
+
+        static double uint32be(const double register1, const double register2) {
+            return ConverterTools::toNumber<uint32_t>(register1, register2);
+        }
+
+        static double uint32le(const double register1, const double register2) {
+            return ConverterTools::toNumber<uint32_t>(register1, register2, true);
+        }
+
+        static double flt32be(const double register1, const double register2) {
+            return ConverterTools::toNumber<float>(register1, register2);
+        }
+
+        static double flt32le(const double register1, const double register2) {
+            return ConverterTools::toNumber<float>(register1, register2, true);
+        }
+
+        static double flt32abcd(const double register1, const double register2) {
+            return ConverterTools::toNumber<float>(register1, register2);
+        }
+
+        static double flt32badc(const double register1, const double register2) {
+            return ConverterTools::toNumber<float>(register1, register2, true);
+        }
+
+        static double flt32cdab(const double register1, const double register2) {
+            return ConverterTools::toNumber<float>(register2, register1);
+        }
+
+        static double flt32dcba(const double register1, const double register2) {
```

Comment:
No need to duplicate flt32le with bacd and flt32be with flt32abcd.

flt32abcd(r1,r2) is the same as flt32cdab(r2,r1) ?

Maybe just add flt32 and flt32be?. The rest depends of correct argument order just like in int32/uint32 case. We can put in docs something like this:
```
flt32(r1,r2): cast to float from abcd bytes array
flt32(r2,r1): cast to float from cdab bytes array
flt32be(r1,r2): cast to float from badc bytes array
flt32be(r2,r1): cast to float from dcba bytes array
```

Docs should specify which register value is high and low pair of float (r1 - high part , r2- low part)

Reply:
OK, done.